shelter/README.md

```diff
@@ -0,0 +1,72 @@
+# shelter
+
+Shelter is designed as a remote attestation tool for customer to verify if their workloads are loaded in a specified intel authorized sgx enclaved.
+The verifying process is as below:
+1. shelter setup a security channel based on mTLS with runE enclaved
+2. runE enclaved will generate/retrieve the quote info of workload running enclave
+3. runE enclaved will get IAS report by quote info from Intel authorized web server
+4. runE enclaved will generate attestation verification report
```

Comment:
Please change runE enclaved to inclavared.

Reply:
ok, got all change requirement about README.md.